fuse/ipns/ipns_unix.go

```diff
@@ -140,7 +140,7 @@ func (s *Root) Lookup(ctx context.Context, name string) (fs.Node, error) {
 	}
 
 	// other links go through ipns resolution and are symlinked into the ipfs mountpoint
-	resolved, err := s.Ipfs.Namesys.Resolve(s.Ipfs.Context(), name)
+	resolved, err := s.Ipfs.Namesys.Resolve(s.Ipfs.Context(), name, 0)
```

Comment:
why is this resolving with depth 0? nothing should resolve infinitely. yeah this is what i want to avoid